genienlp/tasks/almond/utils.py

```diff
@@ -33,7 +33,7 @@ def is_device(token):
     return token[0] == '@'
 
 def process_id(ex):
-    id_ = ex.example_id.rsplit('/', 1)
+    id_ = ex.example_id[0].rsplit('/', 1)
```

Comment:
process_id is called both in BaseAlmondMultiLingualTask with Example and in Iterator when NumericalizedExample is passed. Former is passing a string whereas later passes a list. An if-else statement is needed here to handle it properly.

Reply:
Can you add it? I think the tests crashed with `id_ = ex.example_id.rsplit('/', 1)`

Reply:
added.